contrib/syntax/nano/README.md

```diff
@@ -0,0 +1,29 @@
+Dockerfile.nanorc
+=================
+
+Dockerfile syntax highlighting for nano
+
+Single User Installation
+------------------------
+1. Create a nano syntax directory in your home directory:
+ * `mkdir -p ~/.nano/syntax`
+
+2. Copy `Dockerfile.nanorc` to` ~/.nano/syntax/`
+ * `cp Dockerfile.nanorc ~/.nano/syntax/`
+
+3. Add the following to your `~/.nanorc` to tell nano where to find the `Dockerfile.nanorc` file
+  ```
+## Dockerfile files
+include "~/.nano/syntax/Dockerfile.nanorc"
+  ```
+
+System Wide Installation
+------------------------
+1. Copy `Dockerfile.nanorc` to `/usr/share/nano`
```

Comment:
Sorry, I should've been more clear on this point.  Can you switch this back to `/usr/local/share/nano`?  That was definitely correct for manual install.  `/usr/share/nano` would just be where packages install, but `/usr/local` is explicitly defined as the "sysadmin area" -- packages don't touch that, so it's safe for sysadmins to assume they can build and install there without causing problems with system/package updates.